docs/reference/cli/players.md

```diff
@@ -0,0 +1,605 @@
+
+The `ir` command simulates an IR code. Valid IR codes are defined in the Default.map file.
+
+Example:
+```
+Request: "bd:a5:a9:9b:9d:df ir 768910ef 11073.575<LF>"
+Response: "bd:a5:a9:9b:9d:df ir 768910ef 11073.575<LF>"
+```
+
+***
+## `irenable`
+
+`<playerid> irenable <0|1|?|>`
+
+The `irenable` command enables or disables IR processing for the player on or off. Use `0` to disable, `1` to enable, `?` to query and no parameter to toggle IR processing of the player.
+For remote streaming connections, the command does nothing and the query always returns `1`.
+
+Examples:
+```
+Request: "04:20:00:12:23:45 irenable 1<LF>"
+Response: "04:20:00:12:23:45 irenable 1<LF>"
+
+Request: "04:20:00:12:23:45 irenable ?<LF>"
+Response: "04:20:00:12:23:45 irenable 1<LF>"
+```
+***
+## `connect`
+
+`<playerid> connect <ip>`
+
+The `connect` command tells a Squeezebox 2 or newer player to connect to a different server address.
+
+Supported values are:
+
+>ip - A dotted IP address to connect to.
+
+If the player is currently a member of a sync-group, then all players in the sync-group will be instructed to switch to the new server and re-establish the sync-group.
+
+Example:
+```
+Request: "bd:a5:a9:9b:9d:df connect 192.168.1.10<LF>"
+Response: "bd:a5:a9:9b:9d:df connect 192.168.1.10<LF>"
+```
+
+***
+## `client forget`
+
+`<playerid> client forget`
+
+The `client forget` command deletes the client/player from the server database.
+
+Example:
+```
+Request: "bd:a5:a9:9b:9d:df client forget<LF>"
+Response: "bd:a5:a9:9b:9d:df client forget<LF>"
+```
+
+***
+## `disconnect`
+
+`disconnect <playerid> <ip>`
+
+The `disconnect` command tells a Squeezebox 2 or newer player on another server instance to disconnect from its server and connect to us. This is the opposite of `connect`, where we tell a player connected to us to connect to a different server.
+
+Supported values are:
+
+>ip - A dotted IP address to connect to.
+
+Example:
+```
+Request: "disconnect bd:a5:a9:9b:9d:df 192.168.1.10<LF>"
+Response: "disconnect bd:a5:a9:9b:9d:df 192.168.1.10<LF>"
+```
+
+***
+## `players`
+
+`players <start> <itemsPerResponse>`
+
+The `players` query returns information about all `players` (physical players as well as streaming clients) known by the server.
+
+* `players 0` will return info about all players.
+* `players 3 2` will return information about two players starting with playerindex 3
+* `players 2 1` will return info about the player with playerindex 2
+
+
+**Accepted tagged parameters:**
+
+| Tag | Description |
+|:---|:---|
+| playerprefs | Comma separated list of preference values to return (for each player). |
+
+
+**Returned tagged parameters:**
+
+Results returned as two blocks.
+
+* First a block containing just the count of players
+* Secondly a standard set of data for each player
+    * Info for each player can in addition include values as per the list of preferences included in the optional playerprefs parameter.
+
+| Block | Tag | Description |
```

Comment:
I find this table a bit hard to read, but I am not sure how to make it better...